src/main/scala/com/gu/contentapi/firehose/client/PublicationLogic.scala

```diff
@@ -34,4 +35,13 @@ trait StreamListener {
    */
   def contentTakedown(contentId: String): Unit
 
+  /**
+    * The CAPI now publishes `Atoms` as well as content, which have their own flow.  This event is called whenever an atom is updated
+    *
+    * @param atom
+    */
+
+  def atomUpdate(atom: Atom): Unit
+
+  def atomTakedown(atomId: String): Unit
```

Comment:
CAPI (and therefore crier) doesn't support atom deletion events. Is this a feature you need now?

Reply:
oh, I didn't know that.  No, well, it would be handy to know but it's not essential.  I just saw the "todo" in the code and thought I might as well do that at the same time :-/  Would you rather I delete this or add a comment that the underlying stream does not support it yet?

Reply:
Hmm, since we currently have no plans to implement it I think it's best to leave it out

Reply:
done